pkg/repos/get.go

```diff
@@ -152,37 +145,44 @@ func (m *Manager) deferredSetUpCredentialHelpers(ctx context.Context, cliCfg *co
 		return err
 	}
 
-	var needsBuild bool
+	tool := types.Tool{
+		Source: types.ToolSource{
+			Repo: &types.Repo{
+				Root: runtimeEnv.VarOrDefault("GPTSCRIPT_CRED_HELPERS_ROOT", "https://github.com/gptscript-ai/gptscript-credential-helpers.git"),
+			},
+		},
+	}
+	tag, err := golang.GetLatestTag(tool)
+	if err != nil {
+		return err
+	}
 
+	var needsDownloaded bool
 	// Check the last revision shasum and see if it is different from the current one.
 	lastRevision, err := os.ReadFile(m.credHelperDirs.RevisionFile)
-	if (err == nil && strings.TrimSpace(string(lastRevision)) != repo.Revision) || errors.Is(err, fs.ErrNotExist) {
+	if (err == nil && strings.TrimSpace(string(lastRevision)) != tool.Source.Repo.Root+tag) || errors.Is(err, fs.ErrNotExist) {
 		// Need to pull the latest version.
-		needsBuild = true
-		if err := git.Checkout(ctx, m.gitDir, repo.Root, repo.Revision, filepath.Join(m.credHelperDirs.RepoDir, repo.Revision)); err != nil {
-			return err
-		}
+		needsDownloaded = true
 		// Update the revision file to the new revision.
-		if err := os.WriteFile(m.credHelperDirs.RevisionFile, []byte(repo.Revision), 0644); err != nil {
+		if err = os.WriteFile(m.credHelperDirs.RevisionFile, []byte(tool.Source.Repo.Root+tag), 0644); err != nil {
 			return err
 		}
 	} else if err != nil {
 		return err
 	}
 
-	if !needsBuild {
+	if !needsDownloaded {
 		// Check for the existence of the gptscript-credential-osxkeychain binary.
```

Comment:
nit:

```suggestion
		// Check for the existence of the credential helper binary.
```

(I should have fixed this myself before but evidently missed it.)

Reply:
Done